public/css/settings.css

```diff
@@ -0,0 +1,72 @@
+.settings{
+  width: 100%;
+  float: right;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+.settings .title-userinfo{
+  text-align: center;
+  margin: 60px 0;
+  text-decoration: underline;
+}
+.row-one,
+.row-tow,
+.row-three,
+.row-four,
+.row-five{
```

Comment:
@AhmedAl-Almi Since they all have the same styles, you could probably just have a `.row` class instead of many `.row-(number)` classes